test_runner/regress/test_sharding_service.py

```diff
@@ -843,6 +847,12 @@ def test_sharding_service_heartbeats(
     env = neon_env_builder.init_configs()
     env.start()
 
+    # Default log allow list permitr connection errors, but this test will use error responses on
```

Comment:
nit: s/permitr/permits

Reply:
updated (I was repushing this anyway to get a fresh run before merging)